style.css

```diff
@@ -152,21 +156,203 @@ button:focus {
   width: 218px;
 }
 
-.card-back:hover {
-  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
-  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
-  transition: .8s;
-  }
-
 .card-back,
 .card-front {
   position: absolute;
   -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
 }
 
+.card-back:hover {
+  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transition: .8s;
+}
+
 .card-front {
   transform: rotateY(180deg);
 }
 
+@media (max-width: 1200px) {
+  .table-card__wrap {
+    padding-right: 0;
+    padding-bottom: 0;
+    width: 160px;
+    height: 240px;
+  }
+
+  .card-back {
+    width: 85%;
+  }
+
+  .card-front {
+    width: 85%;
+  }
+
+  .card-loser {
+    width: 100%;
+  }
+  
+  .card {
+    width: 100%;    
+  }
+}
+
+@media (max-width: 1200px) {
+  .card-front {
+    width: 75%;
+  }
+}
+
+@media (max-width: 980px) {
+  .title {
+    font-size: 36px;
+  }
+
+  .text {
+    font-size: 12px;
+  }
+
+  .difficults {
+    font-size: 32px;
+    line-height: 34px;
+    padding-left: 119px;
+  }
+
+  .card-wrap {
+    width: 38%;
+  }
+
+  .start-card {
+    width: 100%;
+  }
+
+  .difficult-check {
+    top: 36px;
+  }
+
+  .difficult:nth-child(1) .difficult-check {
+    top: 10px;
+  }
+}
+
+@media (max-width: 710px) {
+  .title {
+    padding-left: 60px;
+  }
+
+  .text {
+    padding-left: 65px;
+  }
+
+  .difficults {
+    padding-left: 80px;
+    line-height: 32px;
+  }
+
+  .btn {
+    margin-left: 48px;
+  }
+
+  .cards {
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 675px) {
+  .bottom {
+    flex-flow: wrap;
+    margin-top: 53px;
+  }
+
+  .cards {
+    display: none;
+  }
+
+  .table-card__wrap {
+    width: 150px;
+    height: 220px;
+  }
+}
+
+@media (max-width: 610px) {
+  .title {
+    padding-left: 25px;
+  }
+
+  .text {
+    padding-left: 32px;
+  }
+
+  .difficults {
+    padding-left: 37px;
+  }
+
+  .btn {
+    margin-left: 29px;
+  }
+}
+
+@media (max-width: 569px) {
+  .title {
+    padding-left: 10px;
+    font-size: 33px;
+  }
+
+  .text {
+    padding-left: 15px;
+    font-size: 10px;
+  }
+}
+
+@media (max-width: 510px) {
+  .title {
+    text-align: center;
+    padding-left: 0;
+  }
+
+  .text {
+    padding-left: 0;
+    text-align: center;
+    font-size: 15px;
+  }
+
+  .difficults {
+    padding-left: 0;
+    text-align: center;
+  }
+
+  .bottom {
+    justify-content: center;
+  }
+
+  .btn {
+    margin-left: 0;
+  }
+
+  .difficult-check {
+    left: 135px;
+    top: 33px;
+  }
+}
+
+@media (max-width: 476px) {
+  .difficult-check {
+    left: 110px;    
+  }
+
+  .table-card__wrap {
+    width: 110px;
+    height: 150px;    
+  }
+
+  .card-front {
+    width: 70%;    
+  }
+}
 
+@media (max-width: 415px) {
+    .difficult-check {
+      left: 60px;    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/jsFinalTask/pullRequest?prid=6141138)